docs/hbase.md

```diff
@@ -0,0 +1,102 @@
+# `hbase` Metrics Receiver
+
+The `hbase` metric receiver can fetch stats from a HBase server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html). It collects metrics specific to the local region server, as well as metrics presented by the Master node if the node being monitored is indeed the Master.
+
+For High Availability configurations it is recommended to monitor the active Master node only. Monitoring the active Master and its replicants will result in duplicate metrics for the Master node.
+
+## Prerequisites
+
+In order to expose a JMX endpoint, you must set the `com.sun.management.jmxremote.port` system property. It is recommended to also set the `com.sun.management.jmxremote.rmi.port` system property to the same port. To expose JMX endpoint remotely, you must also set the `java.rmi.server.hostname` system property. By default, these properties are set in a Hbase deployment's hbase-env.sh file and the default Hbase installation requires no JMX authentication with JMX exposed locally on 127.0.0.1:8050.
```

Comment:
I don't think this "By default" sentence is accurate - your integration test has to manually enable JMX, and also the default port seems to be :10101 and not :8050.